docs/builtins/add-time.md

```diff
@@ -0,0 +1,32 @@
+## add-time
+Use `add-time` to add a specified number of `seconds` to a given `time`.
+
+### Basic syntax
+
+To add `seconds` to a `time`, use the following syntax:
+
+`(add-time time seconds)`
+
+### Arguments
+
+Use the following arguments to specify the *`time`* to which you want to add *`seconds`* using the `add-time` Pact function.
+
+| Argument | Type | Description |
+| --- | --- | --- |
+| `time` | `time` | Specifies the time to which you want to add `seconds`. |
+| `seconds` | `decimal` or `integer` | Specifies the number of seconds to add to the `time`. |
+
+### Return values
+
+The `add-time` function returns the resulting time after adding the specified `seconds` as a time.
+
+### Examples
+
+The following example adds 15 seconds to a specific time in the Pact REPL:
```

Comment:
`add-time` should be used with things like `seconds`, `minutes` etc. Say I want to add 1 hour, 17 minutes and 5 seconds, I should write something like 
```
(add-time some-time (+ (hours 1) (+ (minutes 17) 5)))
```